src/js/App/Header/AppFilter.js

```diff
@@ -0,0 +1,180 @@
+import React, { useState } from 'react';
+import { Dropdown } from '@patternfly/react-core/dist/js/components/Dropdown/Dropdown';
+import { DropdownToggle } from '@patternfly/react-core/dist/js/components/Dropdown/DropdownToggle';
+import { CaretDownIcon } from '@patternfly/react-icons/dist/js/icons/caret-down-icon';
+import { SearchInput } from '@patternfly/react-core/dist/js/components/SearchInput/SearchInput';
+import { TextContent } from '@patternfly/react-core/dist/js/components/Text/TextContent';
+import { Text, TextVariants } from '@patternfly/react-core/dist/js/components/Text/Text';
+import { Grid } from '@patternfly/react-core/dist/js/layouts/Grid/Grid';
+import { GridItem } from '@patternfly/react-core/dist/js/layouts/Grid/GridItem';
+
+import CogIcon from '@patternfly/react-icons/dist/js/icons/cog-icon';
+import ansible from '../../../../static/images/platform-icons/ansible.svg';
+import openshift from '../../../../static/images/platform-icons/openshift.svg';
+import costManagement from '../../../../static/images/platform-icons/cost-management.svg';
+import migrationsNamespace from '../../../../static/images/platform-icons/migrations-namespace.svg';
+import SAP from '../../../../static/images/platform-icons/SAP.svg';
+import insights from '../../../../static/images/platform-icons/Insights.svg';
+
+import './AppFilter.scss';
+
+const AppFilter = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [filterValue, setFilterValue] = useState('');
+
+  const appList = [
```

Comment:
We should get this list from the `main.yml` file and cache it like the left nav.

Reply:
The file is already fetched, we can just reuse it, at least partially (icons might be missing)